Tests/IsraelHiking.API.Tests/test_files/sample_wwwroot/index.html

```diff
@@ -0,0 +1,11 @@
+﻿<!DOCTYPE html>
```

Comment:
Lat nit-picking, I promise, can this file be inlined into the test file it self - i.e. create a mock for the physical file provider to return a stream that is save as a string in the test class?